vms/authentication/templates/registration/password_reset_form.html

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
you would want to have a `{% extends "vms/base.html" %}` and `{% load i18n %}` here. This would enable to statics to appear on this page which are not currently.